src/styling/ProjectsPage.css

```diff
@@ -1,101 +1,174 @@
 /** Multiple card view **/
 
-.pg-container {
+.projects-page {
   max-width: 1400px;
+  margin: 3rem auto;
+}
+
+.projects-page .flow-images {
+  margin: 1rem 0;
 }
 
 .preview-card {
-  border-radius: 10px;
-  margin: 20px 0px;
-  padding: 5px;
+  border-radius: 16px;
+  margin: 16px 0px;
+  box-shadow: 8px 8px 0px 2px rgba(249,245,227,1);
+  padding: 4px;
+  cursor: pointer;
+  background-color: var(--main-grey);
 }
 
-.urgent-tag {
-  border-radius: 50px;
+.preview-card:hover {
   background-color: var(--main-yellow);
-  font-weight: bold;
   color: var(--main-black);
-  border: transparent;
 }
 
-.card-row {
-  margin-top: 10px;
+.preview-card:hover h5 {
+  color: var(--main-black);
+}
+
+.preview-subtitle {
+  margin-bottom: 16px;
+  font-size: 20px;
+}
+
+.project-card-urgent {
+  border-radius: 8px;
+  border: 2px solid var(--main-blue);
+  font-family: HelveticaNeueBold;
+  color: var(--main-blue);
+  padding: 4px 8px;
+  margin: 4px 0 16px;
+  font-size: 14px;
+  display: inline-block;
 }
 
-.center-div {
-  margin: 0 auto;
+.project-tag-space {
+  height: 46px;
+  display: inline-block;
 }
 
-.overview {
-  margin: 0 auto;
-  width: 90vw;
+.project-more-link {
+  color: var(--main-blue);
 }
 
-.broad {
-  border-radius: 10px;
-  margin: 15px 0px;
-  padding: 10px;
+.project-card-description {
+  line-height: 1.3rem;
+  height: calc(1.3rem * 5);
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
 
 /** Selected card view **/
 
-.description-card {
-  border-radius: 10px;
-  padding: 30px;
-  border: transparent;
-  color: var(--main-black);
+.projects-back {
+  color: var(--main-yellow);
+  cursor: pointer;
+  font-size: 20px;
+  margin: 0 0 32px;
+  display: inline-block;
 }
 
-.project-btn {
-  padding: 20px;
-  border-radius: 50px;
-  background-color: var(--main-blue);
-  margin: 10px;
-  border: transparent;
+.projects-back:hover {
+  text-decoration: underline;
 }
 
-.project-btn:hover {
-  background-color: var(--main-yellow);
+.project-description-card {
+  border-radius: 16px;
+  padding: 24px;
+  background-color: var(--main-grey);
+}
+
+.project-description-close:before,
+.project-description-close:after {
+  content:'';
+  position: absolute;
+  width: 24px;
+  height: 4px;
+  background-color :var(--main-cream);
+  border-radius: 2px;
+  top: 0;
+  cursor: pointer;
 }
 
-.page-container {
-  margin: 30px;
+.project-description-close:before {
+  -webkit-transform:rotate(45deg);
+  -moz-transform:rotate(45deg);
+  transform:rotate(45deg);
+  left: -2px;
 }
 
-.scrolling {
-  height: 90vh;
+.project-description-close:after {
+  -webkit-transform:rotate(-45deg);
+  -moz-transform:rotate(-45deg);
+  transform:rotate(-45deg);
+  right: 2px;
+}
+
+.project-description-close {
```

Comment:
Could we move the close button to be in line with the card title "Sample Website Project #"

Reply:
Also button could use some larger margins, esp if above change is made.
<img width="409" alt="Screen Shot 2020-07-16 at 8 41 16 AM" src="https://user-images.githubusercontent.com/43453743/87672039-52d8dd80-c740-11ea-8f81-ecb5c9e5597d.png">

Reply:
i think I might get rid of the close button altogether on mobile view, I think it'll look weird whereever it is and isn't as necessary when the back link is right above it